pkg/controller/build/buildrequest/machineosbuild.go

```diff
@@ -63,20 +62,14 @@ func (m *MachineOSBuildOpts) validateForHash() error {
 // object. We do this to stabilize the hashing of the name so that whether the
 // value comes from the MachineOSConfig or the OSImageURLConfig, the hash will
 // be same, provided that the value is the same.
-func (m *MachineOSBuildOpts) getMachineOSConfigForHashing() *mcfgv1alpha1.MachineOSConfig {
+func (m *MachineOSBuildOpts) getMachineOSConfigForHashing() *mcfgv1.MachineOSConfig {
 	moscCopy := m.MachineOSConfig.DeepCopy()
-	moscCopy.Spec.BuildInputs.BaseOSImagePullspec = ""
-	moscCopy.Spec.BuildInputs.BaseOSExtensionsImagePullspec = ""
-	moscCopy.Spec.BuildInputs.ReleaseVersion = ""
+	// QOCL:Does this function do anything anymore?
```

Comment:
These fields have been removed, unsure about this function's usage

Reply:
This function can probably go away.